offload/plugins-nextgen/common/src/GlobalHandler.cpp

```diff
@@ -237,6 +237,9 @@ void GPUProfGlobals::dump() const {
   outs() << "========== Data ===========\n";
   for (const auto &ProfData : Data) {
     outs() << "{ ";
+// The ProfData.Name maybe array, eg: NumValueSites[IPVK_Last+1] .
+// If we print out it directly, we are accessing out of bound data.
+// Skip dumping the array for now.
 #define INSTR_PROF_DATA(Type, LLVMType, Name, Initializer)                     \
   if (sizeof(#Name) > 2 && #Name[sizeof(#Name) - 2] == ']') {                  \
     outs() << " [...]";                                                        \
```

Comment:
```suggestion
    outs() << "[...] ";                                                        \
```
The test still fails on my end. It expects a space and a number after `[...]`, and right now the code outputs `[...]0`. Adding the space after the `[...]` fixes this.

Reply:
Oh, you are right. Thanks for the fix!